src/main.cpp

```diff
@@ -2,15 +2,28 @@
 #include "IOManagement.h"
 #include "const.h"
 #include "canPDC.h"
+#include "speed_calc.h"
 
 CANPDC canBus(CAN1, DEF);
 
 void setup() {
   Serial.begin(115200);
   initIO();
+  startSpeedCalculation();
 }
 
 void loop() {
   canBus.sendPDCData();
-  canBus.runQueue(DATA_SEND_PERIOD);
-}
\ No newline at end of file
+  canBus.runQueue(1000);
+
+  Serial.print("RPM: ");
+  Serial.println(rpm);
+  Serial.print("MPH: ");
+  Serial.println(mph);
+
+  for (int i = 0; i < 20; i++) {
+    printf("%d |", previousPulses[i]);
+  }
+
+  printf("\n");
```

Comment:
Now that we know it's working, we can remove the tester code and change it back to `canBus.runQueue(DATA_SEND_PERIOD);`. This way, we won't be merging testing code into `main`.